compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -503,7 +503,7 @@ impl<'a, 'tcx> EncodeContext<'a, 'tcx> {
                                 // Prepend path of working directory onto potentially
                                 // relative paths, because they could become relative
                                 // to a wrong directory.
-                                let working_dir = &self.tcx.sess.working_dir;
+                                let working_dir = &self.tcx.sess.opts.working_dir;
```

Comment:
Should we add a comment saying that it's ok to modify the encoded information because we are adding information that is already tracked in the crate_hash?